galite-tests/src/test/kotlin/org/kopi/galite/tests/form/VBlockTests.kt

```diff
@@ -91,21 +92,29 @@ class VBlockTests : JApplicationTestBase() {
   }
 
   @Test
-  @Ignore("TODO: this doesn't seem to be right")
-  fun selectLookupTest() {
+  fun selectLookupTest_NoMatchingValueInTheTable() {
```

Comment:
I suggest the name:  `test VBlock.refreshLookup with no matching value in the table`